internal/ethapi/api.go

```diff
@@ -55,7 +55,7 @@ import (
 // allowed to produce in order to speed up calculations.
 const estimateGasErrorRatio = 0.015
 
-var errBlobTxNotSupported = errors.New("blob-tx not supported")
+var errBlobTxNotSupported = errors.New("signing blob transactions not supported. Use clef to sign. Submit via eth_sendRawTransaction.")
```

Comment:
Much as I'd like to shill `clef`, I think usage-instructions do not belong in errors. If we're executing in a code-path _very_ close to the user (e.g in the main or thereabouts), we can provide usage information alongside an error-message, but putting it into an `error` is Not The Way, imo

Reply:
Roger that 🫡